### ISSUE TITLE
Issue with Header & Footer Animations

### ISSUE THREAD

**Issue by RexlessTazz** [Puharesource/TitleManager#104]
**Image of my current Header & Footer:**
![preview](https://i.gyazo.com/338d92faf7f20425817447f92c772293.png)

**My header and footer config:**
```YAML
header:
  [5;5;0]&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b█▇▆ \n&fCan't Rankup? \n &e/vote \n &b█▇▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0]&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇█▆ \n&fCan't Rankup? \n &f/&evote \n &b▇█▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0]&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇▆█ \n&fCan't Rankup? \n &e/&fv&eote \n &b▇▆█ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0]&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇█▆ \n&fCan't Rankup? \n &e/v&fo&ete \n &b▇█▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0]&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b█▇▆ \n&fCan't Rankup? \n &e/vo&ft&ee \n &b█▇▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0]&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇█▆ \n&fCan't Rankup? \n &e/vot&fe \n &b▇█▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0]&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇▆█ \n&fCan't Rankup? \n &e/vote \n &b▇▆█ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;5]&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇█▆ \n&fCan't Rankup? \n &e/vote \n &b▇█▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n

footer:
  [5;5;0] \n&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b█▇▆ \n &fIP \n &fS21.MineSpan.com:27005 /nl &b█▇▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0] \n&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇█▆ \n &fIP \n &fS21.MineSpan.com:27005 \n &b▇█▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0] \n&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇▆█ \n &fIP \n &fS21.MineSpan.com:27005 \n &b▇▆█ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0] \n&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇█▆ \n &fIP \n &fS21.MineSpan.com:27005 \n &b▇█▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0] \n&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b█▇▆ \n &fIP \n &fS21.MineSpan.com:27005 \n &b█▇▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0] \n&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇█▆ \n &eI&fP \n &fS21.MineSpan.com:27005 \n &b▇█▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;0] \n&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇▆█ \n &fI&eP \n &fS21.MineSpan.com:27005 \n &b▇▆█ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
  [0;5;5] \n&6«-»&7&m----&6 ☤ &7&m----&6«-»\n &b▇█▆ \n &eIP \n &fS21.MineSpan.com:27005  \n &b▇█▆ \n&6«-»&7&m----&6 ☤ &7&m----&6«-» \n
```

**Issue explanation:**
The `\n` no longer makes new lines and I've tried, `<nl>`, `%nl%` & `{nl}`.

**Comment by Puharesource** [comment]
I am aware of this issue, and I'll be fixing it, though you do not have to include everything in a single animation anymore, because you're now able to mix multiple animations and text together.

**Comment by RexlessTazz** [comment]
Still much easier to have it all in one place. Much, much easier.

**Comment by TrevorSayre** [comment]
Since you're only animating those bars, it would actually be easier to make a single animation file for those bars and include it in the header section of the new config. Then you don't have to repeat yourself, and can edit the text only once instead of 8 times.

**Comment by Puharesource** [comment]
Fixed by 393363b50a65ad80c11f31360facc06a5093182d
